src/services/auth/plugins/password/service.ts

```diff
@@ -57,7 +57,7 @@ export class MemberPasswordService {
     });
   }
 
-  async post(actor: Account, repositories: Repositories, newPassword: string) {
+  async post(actor: AuthenticatedUser, repositories: Repositories, newPassword: string) {
```

Comment:
```suggestion
  async post(account: AuthenticatedUser, repositories: Repositories, newPassword: string) {
```
or `authenticatedUser`